### PR TITLE
cert-rotation: give more time for operators to settle

### PR DESCRIPTION
**Pull request by vrutkovs**

### CODE REVIEW

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.16-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.16-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d

**Comment by vrutkovs** [comment]
/cc @tkashem

We need this one to make sure periodics are stable enough

File: ci-operator/step-registry/gather/cert-rotation/gather-cert-rotation-commands.sh [hunk 1]
```diff
@@ -73,7 +73,7 @@ run-on-first-master "
   oc get csr -o yaml > ${ARTIFACT_DIR}/csrs.yaml
   oc get co -o yaml > ${ARTIFACT_DIR}/cos.yaml
   chown -R core:core ${ARTIFACT_DIR}
-  oc --insecure-skip-tls-verify adm must-gather --timeout=15m --dest-dir=${ARTIFACT_DIR}/must-gather
+  oc --insecure-skip-tls-verify adm must-gather --timeout=15m --dest-dir=${ARTIFACT_DIR}/must-gather || true
```
Comment (mfojtik) [inline ci-operator/step-registry/gather/cert-rotation/gather-cert-rotation-commands.sh:76]: Does this mean we skip failed must-gather?

Comment (vrutkovs) [reply ci-operator/step-registry/gather/cert-rotation/gather-cert-rotation-commands.sh:76]: Right, but I don't think it works as expected. Testing a fix in https://github.com/openshift/release/pull/47478

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-suspend-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-suspend-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Review by dinhxuanvu — APPROVED** [review]
/lgtm
/approve

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-suspend-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Comment by vrutkovs** [comment]
/pj-rehearse pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-shutdown-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-sno-cert-rotation-suspend-90d pull-ci-openshift-cluster-kube-apiserver-operator-release-4.15-e2e-metal-ovn-ha-cert-rotation-suspend-90d

**Pull request merged**